ERCS/erc-7596.md

```diff
@@ -0,0 +1,2756 @@
+---
+eip: 7596
+title: Equity Shares Book-Entry System
+description: An auto-pilot book-entry system of equity shares to form web3.0 capital markets by implementing concept of "Code is Law".
+author: Paul Lee (@paul-lee-attorney) <paul_lee0919@163.com>
+discussions-to: https://ethereum-magicians.org/t/add-erc-equity-shares-book-entry-system/18155
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-01-15
+---
+
+## Abstract
+
+This ERC describes the details of the Equity Shares Book-Entry System (the "**System**"), a proposed protocol and ABI definition that can automatically manage the book-entry records of a company with respect to its equity shares and corporate governance behaviors, in accordance with the consitutional document thereof, so as to enable investors, shareholders, managers, and other stakeholders can directly engage in almost all equity share transactions and corporate governance activities under the automatic control of smart contracts, including share subscription, share transfer, share pledge, share consideration payment, share listing, constitution updates, contracts signing, motion submission, voting, candidates nomination, taking office and resignation etc..
+
+Under the automatic control of smart contracts, many “claim rights” that require cooperation of obligors are transformed into "disposal rights" that can be unilaterally exercised by right holders. In this way, the possibility of defaults will be completely eliminated. So that, the System may fundamentally solve the corporate governance problems such as **"Insider Control"** and **"Misleading Statements"** that has plagued the capital markets for many years, and completely realize:
+1. right holders may **directly exercise** their rights; 
+2. obligors have **no chance** to default; and
+3. **real-time** information disclosure.
+
+By booking the equity shares on EVM blockchain, investors may use cryptocurrency or Central Bank Digital Currencies to pay for share consideration, which means effectively connecting the company’ equity shares to the trillion-dollar crypto markets, so that the companies may have an opportunity to have their equity value reevaluated and reconfirmed with the robust support of the huge liquidity.
```

Comment:
Overall, not a bad abstract. I think it could do with a bit of condensing, and a bit more technical details.